### PR TITLE
First version homepage

### PR DESCRIPTION
**Pull request by Bunnymand**
@PhillipDenmark @karenhyt look at his terrible page and suggest how to go directly to the picture please

### CODE REVIEW

**Comment by PhillipDenmark** [comment]
Excellent piece of work. Can I hire you as a consultant? ;-)

**Comment by Bunnymand** [comment]
I was looking for help to get the picture directly in the homepage, any ideas then we can talk about the consultant part later.......I am growing more expesive by the minute I am sitting here

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
 # Bunnymand.github.io
+
+## What should I do with my project
```
Comment (karenhyt) [inline README.md]: Don't you need a question mark? :smile_cat:

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+[Rescue](http://www.bunnymen.com/)
+
+Can i do **bold**
+
+[And add a picture of the Bunnymen](http://www.bunnymen.com/wp-content/uploads/2012/04/bunnymen-mail.jpg)
```
Comment (PhillipDenmark) [inline README.md]: You need an excalmation mark
text

**Pull request merged**